modules/ROOT/pages/apikit-workflow-convert-existing.adoc

```diff
@@ -1,104 +1,239 @@
-= Workflow: Convert an Existing Mule Project to an APIkit Project
+= Converting a Mule Project to an APIkit Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If you create a new Studio project and do _not_ specify an API definition file,  APIkit generates a skeletal RAML file and XML configuration that includes the following things:
-
-* HTTP Listener configuration
-* APIkit Router component
-* Error handling flows
-
-The following screenshot shows the Specify API Definition File Location or URL option, which you uncheck to create the backend first:
-
-image::backend-first-project.png["The window displays the project name, the runtime server, and the unchecked option in the Specify API Definition File Location or URL option in the API implementation section."]
-
-Later, you can add an API specification, and then use API Console to simulate the API.
-
-== Add APIkit to an Existing Design
-
-You add APIkit components to an existing design, for example a legacy project that you want to modernize using RAML and APIkit, by creating the RAML code within Studio as follows:
-
-*RAML-based API*
-
-* Define the API definition files.
-* Create a main flow, or use an existing one, that includes an HTTP Listener and APIkit Router.
-* Generate backend flows.
-
-The generated backend flows contain the core logic of the API, error handling flows, API Console. You need to add REST resource-action pairing.
-
-== Define the API Definition Files
-
-If you want a RAML-backed API, you can use Studio to create the file: Studio: File > New > RAML API Definition.
-
-image::new-raml.png["Window displays the folder, the name, and the title for creating the new RAML API definition file."]
-
-For example:
-
-. Use the RAML specification and dependent JSON example files provided in the Workflow: Generating a Mule Flow from an API Specification.
-
-. Right-click the name of the RAML in /src/main/resources/api and select Validate. Proceed if there are no problems with the RAML.
-
-. Create any dependent files, or drag and drop them into Studio beside the main RAML. For example, drag the user-example.json to /src/main/resources/api.
+If you chose a backend-first design approach for your Mule project or a Mule application containing business logic, you can convert them into an APIkit project and expose them as a REST service. Fulfill this task following two approaches:
```

Comment:
convert them is plural, but your "or" makes the subject singular.

Reply:
Fulfill this task following two approaches: > Use one of these approaches:

Reply:
Better in the present tense.

Reply:
I like it your way too! Thanks!